Frequency: 30.0

Frame: livox_frame

BackgroundModel_Resolution: 0.1
BackgroundModel_BuildingTime: 60.0
BackgroundModel_Inflate: true
BackgroundModel_Filter_BBX: true

#global coordinate system; x:(+:forward/-:backward) y:(+:left/-:right) z:(+:up/-:down)
BackgroundModel_Filter_BBX_min: [3.0, -20.0, -1.5]
BackgroundModel_Filter_BBX_max: [100.0, 20.0, 15.0]

Threshold_Seconds_For_Background_Filter: 25.0

Topic_Sub_LidarCloud: /livox/lidar
Topic_Pub_FilteredPointCloud: /filtered_cloud
Topic_Pub_BackgroundModel: /background_model